blog/src/User/Console/CreateCommand.php

```diff
@@ -12,14 +12,19 @@
 use Symfony\Component\Console\Output\OutputInterface;
 use Symfony\Component\Console\Style\SymfonyStyle;
 use Throwable;
+use Yiisoft\FormModel\FormHydrator;
```

Comment:
Psalm Level 1 Testing Recommendation: Add the following to support recommendation below: 

use App\User\User;